scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -100,6 +100,9 @@ namespace <%= namespace %> {
             #if UNITY_IOS
             WebCheckout = new iOSWebCheckout(this, client);
             NativeCheckout = new iOSNativeCheckout(State);
+            #else
+            WebCheckout = null;
+            NativeCheckout = null;
```

Comment:
Not in this pr, but it would be cool to use the null object pattern here and have `NullWebCheckout` which responds to the same interface as the iOS/other web/native checkout objects and just no-ops or throws errors